ge06_team04/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Motorcycle Registry : VIN as Serial Number",
+    "summary": "Serail number extension for Motorcycle Registry",
+    "description": """ 
+    Motorcycle Registry : VIN as Serial Number \n
+    This module creates the motorcycle serial number based on the VIN format. \n
+    Task id: 3426902 \n
+    Dev: team4""",
+    "version": "1.0.0",
+    "category": "Kauil/Registry",
+    "license": "OPL-1",
+    "depends": ["motorcycle_registry"],
+    "data": [
+        "data/stock_lot_data.xml",
+    ],
+    "demo": [],
```

Comment:
Don't let unused keys in the manifest,  in this case we can remove demo, remember the less code we write the better :)